### DISCUSSION TITLE
GitHub Copilot requires VS Code 1.57+. Please update your VS Code

### DISCUSSION THREAD

**Discussion by yangjeep** [community/community#18675] [Copilot Conversations] [answered]
```
Version: 1.68.0 (Universal)
Commit: 4af164ea3a06f701fe3e89a2bcbb421d2026b68f
Date: 2022-06-08T11:44:00.250Z (1 wk ago)
Electron: 17.4.7
Chromium: 98.0.4758.141
Node.js: 16.13.0
V8: 9.8.177.13-electron.0
OS: Darwin x64 21.5.0
```

I'm on GitHub Copilot v1.9.5350

I've tried to enable 	
```json
"enable-proposed-api": ["github.copilot"],
```
in argv.json according to some old threads, doesn't seem to work...

Any suggestions?

**Comment by davecheney** [comment, marked as answer]
Please double check you are using VSCode as VScodium or OSSCode are not currently supported. https://github.com/orgs/github-community/discussions/16815

**Reply by yangjeep** [reply]
Thanks!
Ya I confirm it is VSCode... I'm aware that OSSCode has problems.

**Reply by davecheney** [reply]
> I'm on GitHub Copilot v1.9.5350

this is far too old, please upgrade.

**Reply by yangjeep** [reply]
Thanks! It seems like the problem! 
Looks like I had to remove the extension and re-install from the extension marketplace.
